scripts/menus/main_menu.gd

```diff
@@ -18,7 +18,7 @@ var button_click_sfx_instance: PooledAudioStreamPlayer
 func _ready() -> void:
 	# We reserve sfx player instances so that we can play these sounds back monophonically
 	button_hover_sfx_instance = SoundManager.instance("ui", "ui_bloop", "UI Sounds")
-	button_click_sfx_instance = SoundManager.instance("ui", "ui_buzz", "UI Sounds")
+	button_click_sfx_instance = SoundManager.instance("ui", "ui_bloop", "UI Sounds")
```

Comment:
hover and click really shouldnt be the same sound from a ux perspective. not a blocking change though.